byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -14,20 +14,43 @@
 
 package com.google.maps;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceConfig;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.common.flogger.FluentLogger;
 import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
 import com.google.maps.model.PlaceType;
+import com.google.sps.data.Trip;
 import java.io.IOException;
+import java.lang.reflect.Type;
 import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns all PlaceTypes from Places API. */
+/**
+ * Servlet that returns all PlaceTypes from Places API. Stores which placeTypes, or interests, user
+ * selects in datastore with their specific Trip Entity.
+ */
 @WebServlet("/api/places")
 public final class PlacesServlet extends HttpServlet {
 
+  /** {@link Type} of an {@link ArrayList} containing {@link String}, for gson decoding. */
+  private static final Type ARRAYLIST_STRING = new TypeToken<ArrayList<String>>() {}.getType();
+
+  private static final FluentLogger logger = FluentLogger.forEnclosingClass();
```

Comment:
I don't see anywhere where this logger is used here right now, remove?

Reply:
Updated

Reply:
Updated again: used logger when the trip is null on top of setting the response status code